app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'user_id'
+  belongs_to :post
+
+  def self.update_comment_counter(post)
+    post.update(commentsCounter: post.comments.size)
+  end
```

Comment:
- [ ] It would be better if we use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_post_counter method every time a new post has been saved to database, then make the update_post_counter method private

Reply:
Thank you for your suggestion. I implemented this.